packages/dds/tree/src/test/tableSchema.spec.ts

```diff
@@ -295,6 +295,28 @@ describe("TableFactory unit tests", () => {
 				rows: [{ id: "row-0", props: { label: "Row 0" }, cells: {} }],
 			});
 		});
+
+		// Tables manage to make ids readonly at the type level:
```

Comment:
This is actually something I need to change about the table APIs. Tablero actually needs these to be mutable. Or, at least, according to its internal API surface.

Before we promote the table APIs to beta, I need to make the IDs mutable. If you want to tackle that here, please feel free. But if not, I think we should not add this test.

Reply:
If you want to make them mutable, you will want a test for that. Having this test exist, then be updated if/when you make them mutable seems like it would be better than having no test for now.

Mutating the identifiers here seems questionable though. It breaks all existing references to the row/column. Is that actually necessary since you can do roughly the same thing by replacing the row and column nodes with other nodes.

Reply:
> If you want to make them mutable, you will want a test for that. Having this test exist, then be updated if/when you make them mutable seems like it would be better than having no test for now.

Fair point. I'll approve.

> Mutating the identifiers here seems questionable though. It breaks all existing references to the row/column. Is that actually necessary since you can do roughly the same thing by replacing the row and column nodes with other nodes.

Very good point. I need to dig a bit deeper into the requirements here. I was literally writing unit tests for these cases when I saw the notification for this PR. We'll see what ends up actually being needed.